examples/guided.py

```diff
@@ -291,16 +291,8 @@ def perform_installation(mountpoint):
 
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
-<<<<<<< HEAD
 			if archinstall.arguments['mirror-region'] != None:
 				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
-=======
-
-			# Configure the selected mirrors in the installation
-			if archinstall.arguments.get('mirror-region', None):
-				installation.set_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors in the installation medium
```

Comment:
This is from `master` and should stay : )

Reply:
it can't really stay because of how the we return mirrors now